gossip/persistence_test.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2014 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+// Author: Spencer Kimball (spencer.kimball@gmail.com)
+
+package gossip_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/cockroachdb/cockroach/gossip"
+	"github.com/cockroachdb/cockroach/gossip/resolver"
+	"github.com/cockroachdb/cockroach/gossip/simulation"
+	"github.com/cockroachdb/cockroach/util/leaktest"
+)
+
+type testPersistence struct {
+	read, write bool
+	info        gossip.BootstrapInfo
+}
+
+// Read implements the gossip.Persistence interface.
+func (tp *testPersistence) ReadBootstrapInfo(info *gossip.BootstrapInfo) error {
+	tp.read = true
+	*info = tp.info
+	return nil
+}
+
+// Write implements the gossip.Persistence interface.
+func (tp *testPersistence) WriteBootstrapInfo(info gossip.BootstrapInfo) error {
+	tp.write = true
+	tp.info = info
+	return nil
+}
+
+// TestGossipPersistence
```

Comment:
Tests don't need doc comments unless you have something to say (golint doesn't care about test comments)